docs/conf.py

```diff
@@ -185,12 +186,12 @@
 # -- Options for LaTeX output --------------------------------------------------
```

Comment:
If you do a `sphinx-quickstart` in a new project, it produces a very very very small `conf.py` file that doesn't have a lot of these options. I did a pass on some of my libraries to reduce the `conf.py` file. Maybe we do a pass on reducing the `conf.py` file in this PR?

For example, here's the Markus one: https://github.com/willkg/markus/blob/main/docs/conf.py